cranelift/filetests/filetests/isa/x64/call-conv.clif

```diff
@@ -193,8 +193,8 @@ block0(
 ;   movq    %rsp, %rbp
 ;   subq $0x90, %rsp
 ; block0:
-;   movq    rbp(stack args max - 32), %r10
-;   movq    rbp(stack args max - 24), %r11
+;   movq <offset:0>+-0x20(%rbp), %r10
+;   movq <offset:0>+-0x18(%rbp), %r11
```

Comment:
Pre-existing but I'm just noticing it now -- whether in this PR or separately, could we update the printing format for these synthetic amodes so that we don't have "x+-y" to mean "x-y"? (Slightly more complex in the formatting code but hopefully not too bad)